src/main/java/demohometask/demohometask/Match.java

```diff
@@ -0,0 +1,92 @@
+package demohometask.demohometask;
+
+import org.springframework.stereotype.Component;
+
+import java.io.FileOutputStream;
+import java.io.ObjectOutputStream;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Random;
+
+@Component
+public class Match {
+    private static int round;
+    private static int numberOfRounds = 3;
+    private static String score;
+
+
+    public static List<Team> createListOfTeams(){
```

Comment:
Manipulation with teams should be placed in the component that is responsible for managing teams. Please, create separate class for that

Reply:
Created TeamManipulation class.